src/components/notifications/NotificationCard.tsx

```diff
@@ -0,0 +1,74 @@
+import { Text } from "../ui/typography";
+import { StyleSheet, View, Image } from "react-native";
+import Divider from "../ui/Divider";
+import { theme } from "@src/styles/theme";
+import { NotificationType } from "@src/interfaces/notification.types";
+import dayjs from "dayjs";
+import Stack from "../ui/Stack";
+
+const NotificationCard = ({ item }: { item: NotificationType }) => {
+  const diff = dayjs().diff(dayjs(item.createdAt), "day");
+
+  const displayTime = () => {
+    if (diff === 0) {
+      return (item.createdAt = dayjs(item.createdAt).format("h:mm A"));
+    }
+    if (diff === 1) {
+      return "Yesterday";
+    }
+    if (diff > 1) {
+      return dayjs(item.createdAt).format("DD-MM-YYYY");
+    }
+  };
+
+  return (
+    <View>
+      <Stack flexDirection="row" pb={16} pt={16}>
+        {/* TODO: get correct image */}
+        <Image
+          source={require("../../../assets/img/avatar.png")}
+          style={styles.avatar}
+        />
+        <Stack flex={1}>
+          <Text
+            style={styles.notificationTitle}
+            level="footnote"
+            weight="semibold"
+          >
+            {item.title}
+          </Text>
+          <Text style={styles.notificationDetail} level="caption_1">
+            {item.message}
+          </Text>
+        </Stack>
+
+        <Text style={styles.notificationTime} level="caption_1">
+          {displayTime()}
```

Comment:
When the message is long, the time disappears.
![image](https://github.com/LangaraBeetles/weaup-react-native-app/assets/26020262/63017061-9342-4674-9929-71c27a042e09)

Reply:
resolved!
It might have something to do with the data in development.